editor/src/utils/FileSystem.cpp

```diff
@@ -20,7 +20,7 @@ namespace nexo::editor::utils {
         #ifdef _WIN32
             ShellExecuteA(nullptr, "open", folderPath.c_str(), nullptr, nullptr, SW_SHOWDEFAULT);
         #else
-            std::string command = "xdg-open " + folderPath;
+            const std::string command = "xdg-open " + folderPath;
```

Comment:
Add std::quoted to folderPath to prevent command injections (it might still not be foolproof)
```suggestion
            const std::string command = "xdg-open " + std::quoted(folderPath);
```